glowing_waffles/notebooks/acp.py

```diff
@@ -0,0 +1,107 @@
+class ObserveACP:
+    
+    def __init__(self, name, date):
+        
+        from astropy.coordinates import SkyCoord
+        from astropy.time import Time
+        import datetime
+    
+        self.name = name
+        self.filters = []
+        self.binning = 1
+        
+        self.ra = SkyCoord.from_name(name).ra.deg
+        self.dec = SkyCoord.from_name(name).dec.deg
+        
+        #self.date = str(datetime.datetime.today()).split()[0]
+        self.date = date
+        
+        
+    def wait_until(self, time):
+        self.startTime = time
+        
+        
+    def add_filter(self, filt):
+        
+        if filt in self.filters:
+            print("Filter", filt, "is already in list!")
+        
+        elif isinstance(filt, str):
+            # do the string thing
+            self.filters.append(filt)
+        
+        elif isinstance(filt, str) == False:
+            try:
+                # loop over elements in filt
+                for i in filt:
+                    if i in self.filters:
+                        print("Filter", i, "is already in list")
+                        
+                    else:
+                        self.filters.append(i)
+                    
+            except IndexError:
+                raise ValueError('filt must be a non-string iterable')
+                
+        
+    def rm_filter(self, filt):
+        
+        if filt not in self.filters:
+            print("No filter", filt, "in list")
+        
+        elif isinstance(filt, str):
+            self.filters.remove(filt)
+            
+        elif isinstance(filt, str) == False:
+            try:
+                for i in filt:
+                    if i in self.filters:
+                        
+                        self.filters.remove(i)
+                        
+                    else:
+                        print("No filter", filt, "in list")
+            except IndexError:
+                raise ValueError('filt must be a non-string iterable')
+                
+        
+        
+    def expose(self, num):
+        self.exposure = num
+        
+        
+    def count(self, num):
+        self.imCount = num
```

Comment:
snake_case, and also make sure you set `self.im_count` to some default value (like `None`) in the `__init__`.